study-builder/studies/testboston/i18n/en.conf

```diff
@@ -119,8 +119,7 @@
     "s3_privacy_who_item9": "Public health and safety authorities, if we learn information that could mean harm to you or others (such as to make required reports about communicable diseases or about child or elder abuse)",
     "s3_privacy_who_item10": "Other:",
-    "s3_privacy_p2": "Some people or groups who get your identifiable information might not have to follow the same privacy rules that we follow and might use or share your identifiable information without your permission in ways that are not described in this form. For example, we understand that the sponsor of this study may use your identifiable information to perform additional research on various products or conditions, to obtain regulatory approval of its products, to propose new products, and to oversee and improve its products’ performance. We share your identifiable information only when we must, and we ask anyone who receives it from us to take measures to protect your privacy. The sponsor has agreed that it will not contact you without your permission and will not use or share your identifiable information for any mailing or marketing list. However, once your identifiable information is shared outside Partners, we cannot control all the ways that others use or share it and cannot promise that it will remain private.",
-    "s3_privacy_p3": "Because research is an ongoing process, we cannot give you an exact date when we will either destroy or stop using or sharing your identifiable information. Your permission to use and share your identifiable information does not expire.",
+    "s3_privacy_p2": "Some people or groups who get your identifiable information might not have to follow the same privacy rules that we follow and might use or share your identifiable information without your permission in ways that are not described in this form. For example, we understand that the sponsor of this study may use your identifiable information to perform additional research on various products or conditions, to obtain regulatory approval of its products, to propose new products, and to oversee and improve its products’ performance. We share your identifiable information only when we must, and we ask anyone who receives it from us to take measures to protect your privacy. The sponsor has agreed that it will not contact you without your permission and will not use or share your identifiable information for any mailing or marketing list. However, once your identifiable information is shared outside Partners, we cannot control all the ways that others use or share it and cannot promise that it will remain priva
```

Comment:
Is this a typo? Why did we delete `s3_privacy_p3`?

Reply:
A mistake: 20175936